scripts/zones/Castle_Oztroja/npcs/Tebhi.lua

```diff
@@ -7,14 +7,6 @@
 local entity = {}
 
```

Comment:
`local pathNodes =
{
    { x = -136.0000, y = 24.2500, z = -21.0000 },
    { x = -140.0788, y = 24.2500, z = -25.9524 },
    { x = -140.1171, y = 24.2500, z = -37.6438 },
    { x = -140.1195, y = 24.2500, z = -45.9640 },
    { x = -140.1225, y = 24.2500, z = -55.6002 },
}

entity.onSpawn = function(npc)
    npc:initNpcAi()
    npc:setPos(xi.path.first(pathNodes))
    npc:pathThrough(pathNodes, xi.path.flag.PATROL)
end`

Reply:
Those were the captures I took for his pathing - he just runs back and forth -- I have the video somewhere if you'd like it?

Reply:
Yeah that would be great to have as reference

Reply:
Well, apparently I do not have it uploaded to the hxicaptures youtube...